api/src/mailbox.rs

```diff
@@ -1024,7 +1024,6 @@ impl AuthAndStashFlags {
 #[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
 pub struct AuthorizeAndStashReq {
     pub hdr: MailboxReqHeader,
-    pub metadata: [u8; 4],
```

Comment:
metadata is included to be used as the DPE type. We should leave this.

Reply:
Should the code use it somehow?

Reply:
Yes, it should be used the same way it's used for STASH_MEASUREMENT: https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/src/stash_measurement.rs#L80

Reply:
The stashing is marked as TODO I see